vowpalwabbit/gen_cs_example.cc

```diff
@@ -11,28 +11,26 @@
 
 #include "io/logger.h"
 
-namespace logger = VW::io::logger;
-
 namespace GEN_CS
 {
 using namespace VW::LEARNER;
 using namespace CB_ALGS;
 using namespace CB;
 
-
-float safe_probability(float prob)
+float safe_probability(float prob, VW::io::logger& logger)
 {
   if (prob <= 0.)
   {
-    logger::log_warn("Probability {} is not possible, replacing with 1e-3.  Fix your dataset. ", prob);
+    logger.out_warn("Probability {} is not possible, replacing with 1e-3.  Fix your dataset. ", prob);
```

Comment:
oooof the `Fix your dataset` :D Should we replace with: `There seems to be something wrong with the dataset`?

Reply:
Hahah, yeah that's much nicer :)